COMETwebapp/Primitives/Cone.cs

```diff
@@ -71,5 +75,29 @@ public Cone(double x, double y, double z, double radius, double height)
             this.Radius = radius;
             this.Height = height;
         }
+
+        /// <summary>
+        /// Set the dimensions of the <see cref="BasicPrimitive"/> from the <see cref="ElementUsage"/> parameters
+        /// </summary>
+        /// <param name="elementUsage">the <see cref="ElementUsage"/> used for the dimensioning</param>
+        /// <param name="selectedOption">the current <see cref="Option"/> selected</param>
+        /// <param name="states">the <see cref="ActualFiniteState"/> that are going to be used to dimensioning the <see cref="BasicPrimitive"/></param>
+        public override Task SetDimensionsFromElementUsageParameters(ElementUsage elementUsage, Option selectedOption, List<ActualFiniteState> states)

```

Comment:
if none of the logic requires this method to be a `Task`, please refactor it to return `void`